src/app/users/users-achievements/users-achievements.scss

```diff
@@ -28,6 +28,7 @@
 
     .achievement-button {
       grid-area: btn;
+      margin: 0.25rem;
```

Comment:
This also adds left/right margin.  Please set that to 0 with `margin: 0.25rem 0;`